packages/flp-config-sub-generator/src/app/index.ts

```diff
@@ -202,8 +204,8 @@ export default class extends Generator {
             {
                 semanticObject,
                 action,
-                title: keysAdded ? `{{${i18nKeyTitle}}}` : title,
-                subTitle: keysAdded && subTitle ? `{{${i18nKeySubTitle}}}` : subTitle
+                title: keysAdded ? `{{${semanticObject}-${action}.${i18nKeyTitle}}}` : title,
```

Comment:
not sure if there is any recommended best practice of text key format.
mixing `-` and `.` made be wonder 🤔

Reply:
Thank you @devinea 
I think logically this makes sense as the semantic object and task are together.

Reply:
FYI https://github.com/SAP/ui5-manifest/blob/main/schema.json#L413

And the title format schema https://github.com/SAP/ui5-manifest/blob/main/schema.json#L2851

so the `-` and `.` would be supported.

Reply:
Thanks for the links @longieirl